sphinx/apidoc.py

```diff
@@ -161,7 +164,7 @@ def create_package_file(root, master_package, subroot, py_files, opts, subs, is_
 def create_modules_toc_file(modules, opts, name='modules'):
     # type: (List[unicode], Any, unicode) -> None
     """Create the module's index."""
-    text = format_heading(1, '%s' % opts.header)
+    text = format_heading(1, '%s' % opts.header, escape=False)
```

Comment:
I don't know where `opts.header` comes from, but it's not unreasonable that it might contain RST we want to keep.

Everything else in this file is names within code, which should not be interpreted as rst